packages/claim_cascade_batteries/taibom-battery/claim_cascade_config.json

```diff
@@ -1,5 +1,7 @@
 {
-  "inference_output_path": "/home/tony/utils/ClaimCascade/packages/claim_cascade/inference_input_vcs",
-  "policy_output_path": "/home/tony/utils/ClaimCascade/packages/claim_cascade/policy_input_vcs",
-  "claim_cascade_root_keys_path": "/home/tony/utils/ClaimCascade/packages/claim_cascade/root_keys"
+  "claim_cascade_root_keys_path": "/home/ash/Documents/ClaimCascade/packages/claim_cascade//unprocessed/keys",
+  "busy_timeout": 3,
+  "unprocessed_inference_verifiable_credentials_path": "/home/ash/Documents/ClaimCascade/packages/claim_cascade/unprocessed/inference_verifiable_credentials",
+  "unprocessed_policy_verifiable_credentials_path": "/home/ash/Documents/ClaimCascade/packages/claim_cascade/unprocessed/policy_verifiable_credentials",
+  "unprocessed_keys_path": "/home/ash/Documents/ClaimCascade/packages/claim_cascade/unprocessed/keys"
```

Comment:
these should be removed and a readme should be added - especially now this repo is public